### PR TITLE
Completed the catalog grid task

### PR DESCRIPTION
**Pull request by Shevtsooov**
- [DEMO LINK](https://Shevtsooov.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,9 +1,25 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+
+  grid-template-columns: repeat(auto-fill, 200px);
+  gap: 48px;
 
   max-width: 944px;
```
Comment (SofiaLiubchenko) [inline src/styles/_catalog.scss:8]: you can delete max-width property

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -1,9 +1,25 @@
...
   margin: 0 auto;
+  padding-bottom: 48px;
```
Comment (SofiaLiubchenko) [inline src/styles/_catalog.scss:10]: you can delete padding-bottom - there is a 'gap' instead

**Review by SofiaLiubchenko — APPROVED** [review]
good job!

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -1,9 +1,25 @@
...
+
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, 200px);
```
Comment (naziamoff) [inline src/styles/_catalog.scss:13]: would be great to create a variable for that

File: src/styles/_catalog.scss (hunk 2 above)
Comment (naziamoff) [reply src/styles/_catalog.scss:10]: I would say it rather looks like a margin, as it sets empty space for the whole catalog, not for some exact card where gap works just fine

**Review by naziamoff — APPROVED** [review]

**Pull request closed without merging**